lib/osv/models.py

```diff
@@ -331,25 +331,31 @@ def _pre_put_hook(self):
         affected_package.package.purl = purl_helpers.package_to_purl(
             affected_package.package.ecosystem, affected_package.package.name)
 
-    self.project = [
+    self.project = list(set([
```

Comment:
nit: sorted(set( would avoid list and later .sort call ?

Reply:
Just trying to avoid a list copy and doing an in-place .sort() instead.

Reply:
smart, i keep forgetting python is a memory hog!